themes/powerline/powerline.base.bash

```diff
@@ -13,8 +13,10 @@ function __powerline_user_info_prompt {
   local user_info=""
   local color=${USER_INFO_THEME_PROMPT_COLOR}
 
-  if sudo -n uptime 2>&1 | grep -q "load"; then
-    color=${USER_INFO_THEME_PROMPT_COLOR_SUDO}
+  if ${CAN_SUDO}; then
```

Comment:
As discussed in #792, please use a different name for this variable. Please use `THEME_CHECK_SUDO` for this.

Also, please use `if [[ "THEME_CHECK_SUDO" = true ]]` this check.